internal/frontend/templates.go

```diff
@@ -75,6 +78,14 @@ func NewTemplates() (*template.Template, error) {
 		"formatTime": func(tm time.Time) string {
 			return tm.Format("2006-01-02 15:04:05 MST")
 		},
+		"toJSON": func(v interface{}) string {
+			output, _ := json.Marshal(v)
+			return string(output)
+		},
+		"formatPbTime": func(ts *timestamppb.Timestamp) string {
+			t, _ := ptypes.Timestamp(ts)
```

Comment:
I think you can use `ts.AsTime()`

Reply:
Indeed, I don't think I'm using either new func.